addons/mrp_account/views/product_views.xml

```diff
@@ -22,7 +22,7 @@
             <field name="model">product.product</field>
             <field name="priority">4</field>
             <field name="inherit_id" ref="product.product_normal_form_view"/>
-            <field name="groups_id" eval="[(4, ref('mrp.group_mrp_user'))]"/>
+            <field name="groups_id" eval="[(4, ref('mrp.group_mrp_manager'))]"/>
```

Comment:
This change doesn't remove `mrp.group_mrp_user` from the view permissions, so MRP users which are only Inventory users will get a view inheritance error.